stripe:
    binarizer:
        h_min: 0
        h_max: 30

    median_blur:
        ksize: 5

    sides_ratio: 1.5
    approx_diff: 3.0
    min_stripe_length: 10
    min_stripe_width: 10


